src/utils/fetch-loop.tsx

```diff
@@ -99,7 +102,12 @@ class FetchLoopInternal<T = any> {
     let errored = false;
     try {
       const data = await this.fn();
-      globalCache.set(this.cacheKey, data);
+      if (this.setCacheToNull) {
+        globalCache.set(this.cacheKey, data);
```

Comment:
Don't change anything if we allow the cache to be set to null. It's the default behavior